apps/src/applab/VisualizationOverlay.js

```diff
@@ -136,6 +135,10 @@ VisualizationOverlay.prototype.onSvgMouseMove_ = function (event) {
     this.mousePos_ = this.mousePos_.matrixTransform(this.screenSpaceToAppSpaceTransform_);
   }
 
+  if (this.shouldShowCrosshair_()) {
+    this.mouseoverApplabControlId_ = this.getMouseoverApplabControlId_(event.target);
+  }
```

Comment:
Could extract from this PR: This fixes an uncaught exception I encountered in testing where the overlay where ti doesn't get the element ID correctly when running (where we're not going to display the overlay anyway).